meta-sokol-flex-support/recipes-core/meta/archive-release.bb

```diff
@@ -19,11 +19,13 @@ BINARY_ARTIFACTS_COMPRESSION ?= ""
 BINARY_ARTIFACTS_COMPRESSION[doc] = "Compression type for images and downloads artifacts.\
  Available: '.bz2' and '.gz'. No compression if empty"
 
+# TODO: Hardcoding flex-os instead of ${DISTRO}, revert once DISTRO variable is up-to-date as
+# per new brand name
 ARCHIVE_RELEASE_VERSION ?= "${DISTRO_VERSION}"
-MANIFEST_NAME ?= "${DISTRO}-${ARCHIVE_RELEASE_VERSION}-${MACHINE}"
-EXTRA_MANIFEST_NAME ?= "${DISTRO}-${ARCHIVE_RELEASE_VERSION}"
+MANIFEST_NAME ?= "flex-os-${ARCHIVE_RELEASE_VERSION}-${MACHINE}"
```

Comment:
we have changed the distro name. see https://github.com/MentorEmbedded/meta-sokol-flex/blob/scarthgap/meta-sokol-flex-distro/conf/distro/sokol-flex.conf#L6. Should we use Distri Name here or Distro and rename Distro as well. lets have a discussion with Awais on this on renaming DISTRO